### ISSUE TITLE
Add metadata to teiHeader for tlg4170.tlg001.perseus-grc1

### ISSUE THREAD

**Issue by sonofmun** [OpenGreekAndLatin/First1KGreek#630]

**Comment by LindaMcNulty** [comment]
This particular file contains the lives of Pindar as presented in the edition from which the text was pulled on Archive.org; however, the Archive text makes no distinction between the lives of Pindar and the beginning of his odes and the Scholia. Should there be any distinction in our files, then? I have updated all of the header information for the edition accordingly.

**Comment by sonofmun** [comment]
There should be a distinction. If you check the TLG catalog, these are all considered to be different works. Also, it is an interesting question whether to separate out the different parts of this file. If each of Pindar's lives is actually considered a different work, then even each Life should be separated from the others. Whether to do this depends on how the different lives are considered within the scholarly community. The TLG catalog suggests that there may be four different lives here:

```
Vita Ambrosiana: vol. 1, pp. 1-4
Vita Thomana et hypothesis Olympiorum: vol. 1, pp. 4-8
Vita metrica: vol. 1, pp. 8-9
Pentathlon et de novem lyricis: vol. 1, pp. 10-11
```

The Scholia have already been separated out in the tlg5034 folder. 
Remember that one guiding principle of CTS is one work, one file. So if a single volume of text contains more than one work, then we separate that out into different files.
I hope this answered the question.

**Comment by LindaMcNulty** [comment]
Thank you, @sonofmun ! One more question as I separate these out into individual files-- where in the TEI headers should the attribution of "Vita Ambrosiana", "Vita Thomana", etc. go? It is not embedded anywhere in the XML. 

I am going to use the naming scheme of tlg4170.tlg001.perseus-grc1.xml; tlg4170.tlg002.perseus-grc1.xml...

**Comment by sonofmun** [comment]
@LindaMcNulty Put the names of the actual works in the `/TEI/teiHeader/fileDesc/titleStmt/title` element. And make sure add an `xml:lang="lat"` attribute to this element.
As far as naming goes, I would do it a bit differently. Since the TLG qualifies all of these works under the URN of tlg4170.tlg001, I would simply add another number behind that to show it is part of that work. We did this with the individual works of Libanius. So I would suggest the following URNs:
```
tlg4170.tlg0011.1st1K-grc1
tlg4170.tlg0012.1st1K-grc1
tlg4170.tlg0013.1st1K-grc1
tlg4170.tlg0014.1st1K-grc1
```
Does that make sense?

**Comment by LindaMcNulty** [comment]
This issue has been resolved with my pull request #1490 !
